examples/gallery/embellishments/pygmt_logo.py

```diff
@@ -0,0 +1,375 @@
+        # space between letter M and yellow line at the right side
+        # fig.plot(x=[1.6, 1.6], y=[1.5, 1.775], pen=f"10p,{color_bg}")
+        fig.plot(x=[1.6, 1.6], y=[1.5, 2.0], pen=f"10p,{color_bg}")
+        # diagonal lines
+        fig.plot(x=[0.33, 0.90], y=[1.527, 1.00], pen=pen_red)
+        fig.plot(x=[0.90, 1.43], y=[1.00, 1.527], pen=pen_red)
+        # middle pick
+        fig.plot(x=0.9, y=0.9, style="d0.3c", fill=color_red)
+        # vertical lines with small distance to horizontal line of letter G
+        fig.plot(x=[0.285, 0.285], y=[0.30, 1.65], pen=pen_red)
+        fig.plot(x=[1.47, 1.47], y=[0.30, 1.65], pen=pen_red)
+
+        # .............................................................................
+        # letter T
+        # .............................................................................
+        # red curved horizontal line
+        fig.plot(x=0, y=0, style="w4.6c/240/-60+i3.7c", fill=color_red)
+        # vertical endings of curved horizontal line
+        fig.plot(x=[-1.05, -1.05], y=[-1.5, -2.5], pen=f"9p,{color_bg}")
+        fig.plot(x=[1.05, 1.05], y=[-1.5, -2.5], pen=f"9p,{color_bg}")
+        # arrow head as inverse triangle with pen for space to blue circle / hexagon
+        fig.plot(x=0, y=-3.55, style="i1.1c", fill=color_red, pen=f"3p,{color_bg}")
+        # arrow tail
+        fig.plot(x=[0, 0], y=[-2, -3.57], pen=f"12p,{color_red}")
+
+        # margin around shape for black_white in dark_mode
+        # Needed ???
+        if black_white and dark_mode:
+            fig.plot(
+                x=0,
+                y=0,
+                style=f"{symbol}{diameter + diameter_add}c",
+                pen=f"1p,{color_dark}",
+                no_clip=True,
+            )
+
+        # .............................................................................
+        # Save
+        # .............................................................................
+        # fig.show()
+        fig_name = "pygmt_logo"
+        fig.savefig(fname=f"{fig_name}.eps")
+        # print(fig_name)
+
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        # Replot and apply rotation
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        fig = pygmt.Figure()
+        pygmt.config(MAP_FRAME_PEN=no_line)
+        fig.basemap(region=region, projection=f"X{(size + 0.3) * 2}c", frame=no_fill)
+
+        fig.image(
+            imagefile=f"{fig_name}.eps",
+            position=f"jMC+w{size * 2}c",
+            # Rotation around z (vertical) axis placed in the center
+            perspective=f"{angle_rot}+w0/0",
+        )
+
+        # .............................................................................
+        # Save
+        # .............................................................................
+        # fig.show()
```

Comment:
You can avoid this part by adding `perspective=f"{angle_rot}"+w0/0` to the first plotting command (i.e. line 117) and adding `perspective=True` to all other subsequent plotting commands.

Reply:
Updated in commit https://github.com/GenericMappingTools/pygmt/pull/3849/commits/e401feaa7ab24742369e951838e67c31bccf4bbb.